jellyfin-model/build.gradle.kts

```diff
@@ -23,8 +23,15 @@ val sourcesJar by tasks.creating(Jar::class) {
 	from(sourceSets.getByName("main").allSource)
 }
 
+val javadocJar by tasks.creating(Jar::class) {
+	dependsOn(tasks.getByName("dokkaJavadoc"))
+	archiveClassifier.set("javadoc")
+	from("$buildDir/dokka/javadoc")
+}
```

Comment:
Can you somehow de-duplicate this code fragment?

Reply:
Not really, gradle does a lot of magic during build script compilation and moving this caused everything to fail.